### PR TITLE
Router behavior in case of multi-tenant isolation environment

### PR DESCRIPTION
**Pull request by pravisankar**

### CODE REVIEW

**Comment by pravisankar** [comment]
@ramr @adellape PTAL

File: install_config/install/deploy_router.adoc [hunk 1]
```diff
@@ -416,6 +416,17 @@ IP address of a connection to be the NATed IP address of the node, rather than
...
 
+[NOTE]
+====
+On OpenShift cluster with
+link:../../architecture/additional_concepts/sdn.html#multitenant-plugin[multi-tenant
+network isolation], router on a non-default namespace with `--host-network=false`
+option will load all routes in the cluster but routes across the namespaces will not
+be reachable due to network isolation. With `--host-network=true` option, routes
+will bypass the container network and it can access any pod in the cluster.
+If isolation is needed in this case, then don't add routes across the namespaces.
```
Comment (ramr) [inline install_config/install/deploy_router.adoc]: Yeah, this sort of leaves the user hanging with not enough info on how to fix this. We should also document how to set these options https://github.com/openshift/origin/blob/master/pkg/cmd/infra/router/router.go#L46   via the DC config 
```  
$  #  assuming 2 namespaces named default and ns1  
$ oc label namespace default router=my-custom-router  
$ oc label namespace ns1  router=my-custom-router  
$  # and label the routes  
$  oc label route route1 "route=custom"  
$  oc label route route2 "route=custom"  

$ oc edit dc router  -o json   
#  in vi add the following env variables, you can also use jq to script this:  
#   {  "name": "ROUTE_LABELS",  "value": "route=custom" },  
#   { "name": "NAMESPACE_LABELS", "value": "router=my-custom-router"},  
#  :wq  
```   
There's no router option to set them on the router command line as of now but the infra command has support for
it (I am going to say untested though). So just setting environment variables for the container for ```ROUTE_LABELS``` and ```NAMESPACE_LABELS``` should work.  

I am ok with the current PR but I think a follow-on would be needed to add more details.

**Comment by adellape** [comment]
Thanks.

**Comment by vikram-redhat** [comment]
Doesn't require a followup.

**Pull request merged**